src/helpers.ts

```diff
@@ -160,14 +161,19 @@ function checkXpub(xpub: string) {
 
 export function init(
   xpub: string,
-  silent: boolean,
-  quiet: boolean,
+  silent?: boolean,
+  quiet?: boolean,
   currency?: string,
   testnet?: boolean,
   derivationMode?: string,
 ) {
-  configuration.silent = silent;
-  configuration.quiet = quiet;
+  if (typeof silent !== "undefined") {
+    configuration.silent = silent;
+  }
```

Comment:
```suggestion
configuration.silent = !!silent;
```
It's a bit more idiomatic to do it this way in Js I believe

Reply:
Good point. This should work but in this project it is a common approach how to check undefined values. More refactoring will be needed.